tencentcloud/resource_tc_instance.go

```diff
@@ -104,6 +104,16 @@ func resourceTencentCloudInstance() *schema.Resource {
 				ForceNew:     true,
 				ValidateFunc: validateInstanceType,
 			},
+			"hostname": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+			},
+			"project_id": &schema.Schema{
```

Comment:
project_id can be modified, it should not be ForceNew

Reply:
Sure, fixed.